### PR TITLE
Opendistro Release 1.9.0

### PR DESCRIPTION
**Pull request by joshuali925**
*Issue #, if available:*
#531 

*Description of changes:*
- Elasticsearch 7.8.0 compatibility
    - Accommodate API changes from elasticsearch 7.8.0
    - Add an additional license in build.gradle for license header check
    - Disable validateNebulaPom

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by zhongnansu** [comment]
Since you modified 30 files, please add more description to this PR, which will help reviewers to review.

File: src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ private void async(Client client, Map<String, String> params, QueryAction queryA
 
         // Preserve context of calling thread to ensure headers of requests are forwarded when running blocking actions
         threadPool.schedule(
-                threadPool.preserveContext(LogUtils.withCurrentContext(runnable)),
+                LogUtils.withCurrentContext(runnable), //TODO: confirm if preserve does happen inside schedule() method
```
Comment (dai-chen) [inline src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java:146]: Please help remove the TODO comment.

**Comment by dai-chen** [comment]
And also please check previous PR if release note should be included here or a separate PR.

File: src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java (hunk 1 above)
Comment (joshuali925) [reply src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java]: Got it.

**Pull request merged**